common/sagas/transactions.ts

```diff
@@ -34,6 +49,38 @@ export function* fetchTxData(action: FetchTransactionDataAction): SagaIterator {
   yield put(setTransactionData({ txhash, data, receipt, error }));
 }
 
+export function* saveBroadcastedTx(action: BroadcastTransactionQueuedAction) {
+  const txBuffer = action.payload.serializedTransaction;
```

Comment:
You could do variable reassigning in the object deconstruction here instead 
```ts
const { serializedTransaction: txBuffer, txIdx: indexingHash } = action.payload
```

Reply:
Oh cool, didn't know you could rename with destructures.